packages/components/src/assets/ItemProps.ts

```diff
@@ -0,0 +1,3 @@
+export type ItemProps = {
```

Comment:
🆒  Should we adopt this approach for the other icon assets too?

Reply:
I think it's a good idea! :)

Reply:
is it only for icons? can be `IconProps` instead?

Reply:
created the task [here](https://vtex-dev.atlassian.net/browse/FS-858?atlOrigin=eyJpIjoiZDM1Zjc5NGUwNjUxNDNjOThkYTQ3NmU5YmMyNGRjYzMiLCJwIjoiaiJ9) ;)